Tests/SwiftDocCTests/Test Bundles/AvailabilityBundle.docc/MyClass.md

```diff
@@ -0,0 +1,7 @@
+# ``MyKit/MyClass``
+
+@Metadata {
+    @Available(iOS, introduced: "16.0")
```

Comment:
Should we include some tests with some more complicated combinations of `@Available`? I'm worried we're mostly covering the happy-path here.

Reply:
How complicated are we thinking? I'm testing that using multiple directives parses correctly in `MetadataAvailabilityTests.testValidDirective`, even if it doesn't test that they show up in the resulting rendering. Should i add more test data to that effect?

Reply:
I _think_ so – the parsing is one thing but there's some specific logic for merging the different availabilities that is covered there, right? Maybe I'm missing something there.